html/pfappserver/lib/pfappserver/PacketFence/Controller/Config/Fingerbank/Settings.pm

```diff
@@ -128,6 +129,17 @@ sub index :Path :Args(0) :AdminRole('FINGERBANK_READ') {
     $c->response->status($status);
 }
 
+sub update_p0f_map :Local :Args(0) :AdminRole('FINGERBANK_UPDATE') {
```

Comment:
Shouldn't we use the same logic as for the database operations ? (using async API calls to avoid slow update timing out on the GUI ?)
https://github.com/julsemaan/packetfence/blob/feature/tcp-fingerprints/html/pfappserver/lib/pfappserver/PacketFence/Controller/Config/Fingerbank/DB.pm#L26

Which make me think that moving subs of that class (DB) here could be a good idea too.